timely/src/dataflow/channels/pact.rs

```diff
@@ -14,37 +14,36 @@ use crate::communication::{Push, Pull, Data};
 use crate::communication::allocator::thread::{ThreadPusher, ThreadPuller};
 use crate::Container;
 
-use crate::worker::AsWorker;
 use crate::dataflow::channels::pushers::Exchange as ExchangePusher;
+use crate::dataflow::ScopeParent;
 use super::{BundleCore, Message};
 
 use crate::logging::{TimelyLogger as Logger, MessagesEvent};
-use crate::progress::Timestamp;
 
 /// A `ParallelizationContractCore` allocates paired `Push` and `Pull` implementors.
-pub trait ParallelizationContractCore<T, D> {
+pub trait ParallelizationContractCore<G: ScopeParent, D> {
```

Comment:
This looks like a regression. Can we leave it as `T`?

Reply:
Yes, just fixed that